src/ripple/protocol/impl/STArray.cpp

```diff
@@ -53,7 +49,7 @@ STArray::STArray (int n)
 STArray::STArray (SField const& f)
     : STBase (f)
 {
-    v_.reserve(reserveSize);
+
```

Comment:
This should now be `= default` in the header.

Reply:
So if I do: `STArray(SField const& f) = default` then the compiler will _automatically_ find the `STBase::STBase(SField const& f)` constructor and call it?

Reply:
Wow, I completely misread this!  For some reason I thought I was talking about the `STArray` default constructor.  Please ignore.  Sorry for the noise.

But adding `explicit` does sound like a good idea.